step.sh

```diff
@@ -94,9 +94,17 @@ echo_details "* url: $url"
 
 echo_info "react-native version:"
 
-REACT_NATIVE_BIN="react-native"
-if [ ! -z "${binary_path}" ] ; then
-    REACT_NATIVE_BIN="${binary_path}/react-native"
+# Find path to the react-native CLI
+declare REACT_NATIVE_BIN
+if [[ -n "$binary_path" ]]; then
+	# If binary_path is specified, use that
+	REACT_NATIVE_BIN="$binary_path/react-native"
+elif output=$(npx which react-native); then
```

Comment:
In an interactive shell, I get the following prompt when `which` is not installed:

```
❯ npx which react-native
Need to install the following packages:
  which@3.0.0
Ok to proceed? (y)
```

Does this automatically installs the package in a non-interactive shell or do we need to provide something like a `-y` flag?

Reply:
Whoops! Good catch. Let me see if I can find a good solution for that.

- [ ] Find solution for needing to install `which` npm package